include/enotify.php

```diff
@@ -84,7 +84,7 @@ function notification($params)
 	// with $params['show_in_notification_page'] == false, the notification isn't inserted into
 	// the database, and an email is sent if applicable.
 	// default, if not specified: true
-	$show_in_notification_page = ((x($params, 'show_in_notification_page'))	? $params['show_in_notification_page']:true);
+	$show_in_notification_page = isset($params['show_in_notification_page']) ? $params['show_in_notification_page'] : true;
```

Comment:
wouldn't `defaults($params, ''show_in_notification_page', true)` be better?

Reply:
Not in this case, `defaults()` only work when we are expecting a truthy value, otherwise the falsy default value is used. However, if we are expecting a falsy value, `defaults()` would use the default value instead.

This is because `defaults()` is based on `x()`.

Reply:
Noticed for some `boot.php` refactor action ;-)